app/constants/currencies.js

```diff
@@ -0,0 +1,153 @@
+  { value: "METIS", label: "Metis" },
+  { value: "DEV", label: "Moonbase Alpha Testnet" },
+  { value: "GLMR", label: "Moonbeam" },
+  { value: "MOVR", label: "Moonriver" },
+  { value: "XNO", label: "Nano" },
+  { value: "CANTO", label: "Native Canto" },
+  { value: "EVMOS", label: "Native Evmos" },
+  { value: "NEAR", label: "NEAR Protocol" },
+  { value: "NAS", label: "Nebulas" },
+  { value: "NEO", label: "Neo" },
+  { value: "CKB", label: "Nervos Network" },
+  { value: "NTRN", label: "Neutron" },
+  { value: "NTRN", label: "Neutron Testnet" },
+  { value: "NIM", label: "Nimiq" },
+  { value: "USDC", label: "Noble" },
+  { value: "NULS", label: "NULS" },
+  { value: "ROSE", label: "Oasis Network" },
+  { value: "OKT", label: "OKX Chain" },
+  { value: "OKT", label: "OKXChain Mainnet" },
+  { value: "ONT", label: "Ontology" },
+  { value: "OETH", label: "Optimistic Ethereum" },
+  { value: "ETH", label: "Optimism Goerli Testnet" },
+  { value: "OSMO", label: "Osmosis" },
+  { value: "XPRT", label: "Persistence" },
+  { value: "LAT", label: "PlatON" },
+  { value: "POA", label: "POA" },
+  { value: "DOT", label: "Polkadot" },
+  { value: "MATIC", label: "Polygon" },
+  { value: "MATIC", label: "Polygon Mumbai Testnet" },
+  { value: "ETH", label: "Polygon zkEVM" },
+  { value: "QTUM", label: "Qtum" },
+  { value: "QSR", label: "Quasar" },
+  { value: "RVN", label: "Ravencoin" },
+  { value: "XRP", label: "XRP" }, // not "ripple"
+  { value: "RON", label: "Ronin" },
+  { value: "SCRT", label: "Secret" },
+  { value: "ETH", label: "Sepolia Testnet" },
+  { value: "BCH", label: "smartBCH" },
+  { value: "BNB", label: "BNB Smart Chain" },
+  { value: "SOL", label: "Solana" },
+  { value: "SOMM", label: "Sommelier" },
+  { value: "STARS", label: "Stargaze" },
+  { value: "STEEM", label: "Steem" },
+  { value: "XLM", label: "Stellar" },
+  { value: "STRD", label: "Stride" },
+  { value: "SUI", label: "Sui" },
+  { value: "TORI", label: "Teritori" },
+  { value: "LUNC", label: "Terra Classic" },
+  { value: "LUNA", label: "Terra" },
+  { value: "XTZ", label: "Tezos" },
+  { value: "tFTM", label: "Fantom Testnet" },
+  { value: "THETA", label: "THETA" },
+  { value: "RUNE", label: "THORChain" },
+  { value: "TT", label: "ThunderCore (TT)" },
+  { value: "TOMO", label: "TomoChain" },
+  { value: "TON", label: "The Open Network" },
+  { value: "TRX", label: "TRON" },
+  { value: "UMEE", label: "Umee" },
+  { value: "VET", label: "VeChain" },
+  { value: "VIA", label: "Viacoin" },
+  { value: "WAN", label: "Wanchain" },
+  { value: "WAVES", label: "Waves" },
+  { value: "xDAI", label: "Gnosis Chain" },
+  { value: "XDC", label: "XinFin Network" },
+  { value: "ZEC", label: "Zcash" },
+  { value: "FLUX", label: "Flux" }, // not "zelcash" ?
+  { value: "ZIL", label: "Zilliqa" },
+  { value: "ETH", label: "ZkSync Era" },
+  // Update crypto-currencies list if needed
+  { value: "not_specified", label: "Not Specified" },
+  ];
```

Comment:
We need a curated list of tokens at least for Ethereum, Polygon and Gnosis chain.

To add the address you can write a helper or something like this.